Konard/Basic-dnd/B126-Race-Steel-Defender.xml

```diff
@@ -0,0 +1,1217 @@
+		<compendium display="false" />
+		<supports>Defender Upgrades</supports>
+		<requirements>ID_B126_DEFENDER_UPGRADES_UPGRADED_BUILD_3</requirements>
+		<prerequisite>6th-level artificer</prerequisite>
+		<description>
+			<p><em>Prerequisites: 6th-level artificer</em></p>
+			<p>You improve upon the foundation of your steel defender's build. Choose one ability score from Strength, Dexterity, Constitution, or Wisdom. The defender's score for the chosen ability increases 2, up to a maximum of 20. You can take this upgrade multiple times.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Class Feature" name="Upgraded Foundation" supports="Upgraded Foundation" number="1" />
+		</rules>
+	</element>
+	<element name="Upgraded Build" type="Class Feature" source="Basic's Box O' Fixes" id="ID_B126_DEFENDER_UPGRADES_UPGRADED_BUILD_5" >
+		<compendium display="false" />
+		<supports>Defender Upgrades</supports>
+		<requirements>ID_B126_DEFENDER_UPGRADES_UPGRADED_BUILD_4</requirements>
+		<prerequisite>6th-level artificer</prerequisite>
+		<description>
+			<p><em>Prerequisites: 6th-level artificer</em></p>
+			<p>You improve upon the foundation of your steel defender's build. Choose one ability score from Strength, Dexterity, Constitution, or Wisdom. The defender's score for the chosen ability increases 2, up to a maximum of 20. You can take this upgrade multiple times.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Class Feature" name="Upgraded Foundation" supports="Upgraded Foundation" number="1" />
+		</rules>
+	</element>
+	<element name="Upgraded Build" type="Class Feature" source="Basic's Box O' Fixes" id="ID_B126_DEFENDER_UPGRADES_UPGRADED_BUILD_6" >
+		<compendium display="false" />
+		<supports>Defender Upgrades</supports>
+		<requirements>ID_B126_DEFENDER_UPGRADES_UPGRADED_BUILD_5</requirements>
+		<prerequisite>6th-level artificer</prerequisite>
+		<description>
+			<p><em>Prerequisites: 6th-level artificer</em></p>
+			<p>You improve upon the foundation of your steel defender's build. Choose one ability score from Strength, Dexterity, Constitution, or Wisdom. The defender's score for the chosen ability increases 2, up to a maximum of 20. You can take this upgrade multiple times.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Class Feature" name="Upgraded Foundation" supports="Upgraded Foundation" number="1" />
+		</rules>
+	</element>
+	
+	<!-- upgraded build strength increases -->
+	<element name="Strength" type="Class Feature" source="Basic's Box O' Fixes" id="ID_B126_DEFENDER_UPGRADES_FOUNDATION_STRENGTH" >
+		<compendium display="false" />
+		<supports>Upgraded Foundation</supports>
+		<description>
+			<p>This upgrade improves the power delivery systems to the joints and increses their power output.</p>
+		</description>
+		<sheet>
```

Comment:
In this case use can do this instead of having duplicate elements

```suggestion
		</description>
		<setters>
			<set name="allow duplicate">true</set>
		</setters>
		<sheet>
```